### ISSUE TITLE
Error with Craft 3.4.11

### ISSUE THREAD

**Issue by ursbraem** [verbb/knock-knock#22]
### Description

After updating craft to 3.4.11, when running the command line `craft` command, I get the following error:

```
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getAbsoluteUrl()'

in /var/www/html/craft/vendor/yiisoft/yii2/base/Component.php:300

Stack trace:
#0 /var/www/html/craft/vendor/verbb/knock-knock/src/KnockKnock.php(72): yii\base\Component->__call('getAbsoluteUrl', Array)
#1 [internal function]: verbb\knockknock\KnockKnock->verbb\knockknock\{closure}(Object(yii\base\Event))
#2 /var/www/html/craft/vendor/yiisoft/yii2/base/Event.php(312): call_user_func(Object(Closure), Object(yii\base\Event))
#3 /var/www/html/craft/vendor/yiisoft/yii2/base/Component.php(636): yii\base\Event::trigger('craft\\services\\...', 'afterLoadPlugin...', Object(yii\base\Event))
#4 /var/www/html/craft/vendor/craftcms/cms/src/services/Plugins.php(272): yii\base\Component->trigger('afterLoadPlugin...')
#5 /var/www/html/craft/vendor/craftcms/cms/src/base/ApplicationTrait.php(1389): craft\services\Plugins->loadPlugins()
#6 /var/www/html/craft/vendor/craftcms/cms/src/console/Application.php(59): craft\console\Application->_postInit()
#7 /var/www/html/craft/vendor/yiisoft/yii2/base/BaseObject.php(109): craft\console\Application->init()
#8 /var/www/html/craft/vendor/yiisoft/yii2/base/Application.php(206): yii\base\BaseObject->__construct(Array)
#9 /var/www/html/craft/vendor/yiisoft/yii2/console/Application.php(89): yii\base\Application->__construct(Array)
#10 /var/www/html/craft/vendor/craftcms/cms/src/console/Application.php(48): yii\console\Application->__construct(Array)
#11 [internal function]: craft\console\Application->__construct(Array)
#12 /var/www/html/craft/vendor/yiisoft/yii2/di/Container.php(400): ReflectionClass->newInstanceArgs(Array)
#13 /var/www/html/craft/vendor/yiisoft/yii2/di/Container.php(159): yii\di\Container->build('craft\\console\\A...', Array, Array)
#14 /var/www/html/craft/vendor/yiisoft/yii2/BaseYii.php(365): yii\di\Container->get('craft\\console\\A...', Array, Array)
#15 /var/www/html/craft/vendor/craftcms/cms/bootstrap/bootstrap.php(245): yii\BaseYii::createObject(Array)
#16 /var/www/html/craft/vendor/craftcms/cms/bootstrap/console.php(43): require('/var/www/html/c...')
#17 /var/www/html/craft/craft(21): require('/var/www/html/c...')
#18 {main}
```

When I uninstall knock-knock, it is gone.
What could that be? The `getAbsoluteUrl` method should still be available in Craft...

### Additional info

- Plugin version: 1.2.4
- Craft version: 3.4.11

**Comment by engram-design** [comment]
Fixed in [1.2.4.1](https://github.com/verbb/knock-knock/releases/tag/1.2.4.1)

**Comment by ursbraem** [comment]
Awesome, thanks!
